Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,94 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+from mentii import user_ctrl
+
+class MentiiAuthentication:
+
+  def __init__(self):
+    return
+
+  @staticmethod
+  def build_user_object(userData):
+    '''
+    Builds a standard user object to return to the client for all authenticated calls
+    '''
+    return {'email': userData['email']}
```

Comment:
you don't need a try catch if you just do a check for that attribute in the space before
if email in userData:
  return whats there
else:
  return something bad

I'd be for that change